messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerService.java

```diff
@@ -161,16 +169,12 @@ public synchronized void start(final StartContext context) throws StartException
                     ROOT_LOGGER.aioInfo();
                 }
                 configuration.setJournalType(JournalType.NIO);
+            } else if (! AIOSequentialFileFactory.isSupported(configuration.getJournalDirectory())) {
```

Comment:
Ok, no problem. I'm not sure that it really helps to have 3 different INFO log related to AIO. At the end of the day, "AIO is not supported by the OS file system, switching to NIO" looks to cover all cases :)